src/style.css

```diff
@@ -1,3 +1,144 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+@media screen and (max-width: 1024px) {
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
+  }
+
+  header a.logo {
+    display: flex;
+    align-items: center;
+  }
+
+  ul {
+    display: flex;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  li {
+    margin-right: 20px;
+  }
+
+  li:first-child {
+    margin-left: 0;
+  }
+
+  li:last-child {
+    margin-right: 0;
+  }
+
+  a {
+    text-decoration: none;
+    text-transform: uppercase;
+    color: black;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    position: relative;
+  }
+
+  a:hover {
+    color: #00acdc;
+  }
+
+  a.is-active {
+    position: relative;
+  }
+
+  a.is-active::after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 4px;
+    background-color: #00acdc;
+    position: absolute;
+    bottom: 0;
+    border-radius: 8px;
+  }
+
+  a.is-active,
+  a:active {
+    color: #00acdc;
+    border-bottom: none;
+    border-radius: 0;
+  }
+}
```

Comment:
The media query for max-width: 1024px is used to apply styles for smaller screens. Ensure that the styles within this block are necessary and do not conflict with other styles.